tests/trainer/logging_tests/test_train_loop_logging_1_0.py

```diff
@@ -771,3 +771,34 @@ def on_train_epoch_end(self, *_):
     trainer.fit(model)
     assert model.epoch_end_called
     assert model.on_train_epoch_end_called
+
+
+def test_unbalanced_logging_with_multiple_optimizers(tmpdir):
```

Comment:
shall we rather have a special test/file for multi-optimizer cases?

Reply:
Why not, let me make the change.